mxcubeweb/core/components/sampleview.py

```diff
@@ -384,8 +384,11 @@ def start_manual_centring(self):
                     "Aborting current centring ..."
                 )
                 HWR.beamline.diffractometer.cancel_centring_method(reject=True)
-
-            logging.getLogger("user_level_log").info("Centring using 3-click centring")
+            msg = (
+                "Centring using %s centring"
+                % HWR.beamline.diffractometer.MANUAL3CLICK_MODE.lower()
+            )
+            logging.getLogger("user_level_log").info(msg)
```

Comment:
This is kind of the worst of all the logging solutions available to us :D

Also I do not see the point of the `lower()` call here.

My personal preference is something like this:

```suggestion
            msg = "Centring using %s centring"
            logging.getLogger("user_level_log").info(msg, HWR.beamline.diffractometer.MANUAL3CLICK_MODE)
```

But seems like the general consensus is to use f-strings:

```suggestion
            msg = f"Centring using {HWR.beamline.diffractometer.MANUAL3CLICK_MODE} centring"
            logging.getLogger("user_level_log").info(msg)
```

Reply:
I'm in the f-string team, but when I committed the changes, it seemed like ruff didn’t like them, turns out I was wrong (I just tried it again now).

I find f-strings more readable, but yeah, this:
```python
msg = "Centring using %s centring"
logging.getLogger("user_level_log").info(msg, HWR.beamline.diffractometer.MANUAL3CLICK_MODE)
```
can be a cleaner solution when you’re dealing with long variable names.

I guess I’ll just decide case by case, depending on the length :)

...about the `lower()` I just wanted to maintain the case